src/rpc/methods/eth/filter/mod.rs

```diff
@@ -421,6 +421,78 @@ impl EthEventHandler {
 
         Ok(collected_events)
     }
+
+    pub async fn eth_get_events_for_event_filter<DB: Blockstore + Send + Sync + 'static>(
```

Comment:
Are we able to contrive some unit tests around it?

Reply:
this method is removed